shotover-proxy/tests/runner/observability_int_tests.rs

```diff
@@ -35,20 +37,32 @@ shotover_chain_messages_per_batch_count{chain="redis source",quantile="0.999"}
 shotover_chain_messages_per_batch_count{chain="redis source",quantile="1"}
 shotover_chain_total_count{chain="redis source"}
 shotover_query_count{name="redis-chain"}
+shotover_sink_to_source_latency_seconds_count{source="redis"}
+shotover_sink_to_source_latency_seconds_sum{source="redis"}
+shotover_sink_to_source_latency_seconds{source="redis",quantile="0"}
+shotover_sink_to_source_latency_seconds{source="redis",quantile="0.1"}
```

Comment:
Why 0.1 quantile? Trying to figure out why I would use this vs average (0.5) or the higher ones. Could be useful to try to figure out if I'm on a long tail distribution, but not sure how.

Reply:
I added it to help debug strange quantile behavior.
I described it under:
> Some of the histograms are giving really wonky results, percentiles 0.1 through to 0.999 are all the same value.

in the main PR description.

Maybe I should remove it again, but I figured if it was useful now it'll likely be useful later.